src/api/sales.ts

```diff
@@ -0,0 +1,44 @@
+import {API_BASE_URL} from '@env';
+import {createClient} from '@atb/api/client';
+import type {
+  SalesTripPatternLeg,
+  OfferFromLegsResponse,
+  Traveller,
+} from './types/sales';
+
+export const client = createClient(API_BASE_URL);
+const tripPatternEndpoint = `/sales/v1/search/trip-pattern`;
+
+// TODO: Should this be a combined function for multiple offers?
```

Comment:
I guess the answer to this is no, after the BFF takes care of multi-trip-pattern lookup.

```suggestion
```